dags/benk/generate.py

```diff
@@ -40,4 +45,4 @@
             prepare = Prepare(definition.catalog)
 
             # Add Prepare
-            cross_downstream(prepare.get_leaf_nodes(), flatten_list([i.get_start_nodes() for i in imports]))
+            chain(prepare.get_leaf_nodes(), *initialise.get_start_nodes())
```

Comment:
Same here, you're chaining all possible start nodes, while they should be on the same level.

Say you have prepare leaf nodes L1, L2, and initialise start nodes i1 and i2.
What you do now:

[L1, L2] -> i1 -> i2
The start nodes are depending on each other here, while they should start at the same time: [L1, L2] -> [i1, i2]

Again, no big issue when here is only one start node in initialise (which is the case at this moment), but it doesn't agree with the get_start_nodes() signature.

`cross_downstream(prepare.get_leaf_nodes(), initialise.get_start_nodes())`